Kitodo/src/main/java/org/kitodo/production/services/dataeditor/DataEditorService.java

```diff
@@ -136,13 +136,15 @@ public static List<SelectItem> getAddableMetadataForGroup(Ruleset ruleset, TreeN
         ProcessFieldedMetadata fieldedMetadata = ((ProcessFieldedMetadata) metadataNode.getData());
         ComplexMetadataViewInterface metadataView = fieldedMetadata.getMetadataView();
         List<SelectItem> addableMetadata = new ArrayList<>();
-        for (MetadataViewInterface keyView : metadataView.getAddableMetadata(fieldedMetadata.getChildMetadata(),
-                fieldedMetadata.getAdditionallySelectedFields())) {
-            addableMetadata.add(
-                    new SelectItem(keyView.getId(), keyView.getLabel(),
-                            keyView instanceof SimpleMetadataViewInterface
-                                    ? ((SimpleMetadataViewInterface) keyView).getInputType().toString()
-                                    : "dataTable"));
+        if (Objects.nonNull(metadataView)) {
```

Comment:
Wondering why `metadataView` can be `null`. This looks like a bug elsewhere.

Reply:
Yes, I think you are correct. This pull request is not meant to fix the underlying problem of the metadata handling but is just intended to better deal with the resulting null pointer exception.